babel-preset/plugins/__tests__/rewrite-modules-test.js

```diff
@@ -33,6 +33,20 @@ describe('rewrite-modules', function() {
         expect(result.code).toEqual('require(\'test/test\');');
       });
 
+      it('should replace the prefix on type imports', function() {
+        let result = babel.transform(
+          'import type Test from \'test\';',
+          {
+            plugins: [
+              "syntax-flow",
```

Comment:
And then single quotes here (lint warns about this but just a warning so doesn't show up in travis, sorry)